src/orient/viz/rational-calc.notebook.js

```diff
@@ -121,12 +120,12 @@ md`#### Benchmarks`
 
 bench = ({
   "column_leaf_hash_time": 1.7028e-5/2,
-  "kdf_time": 5.4e-7/2,
+  "kdf_time": 5.4e-7,
   "merkle_tree_datahash_time": 1.28e-8/2,
   "merkle_tree_hash_time": 1.7028e-5/2,
   "snark_constraint_time": 3.012e-5/2,
   "ticket_hash": 1.7028e-5/2,
-  "kdf_time_fastcpu": 0.0000000128/2
+  "kdf_time_fastcpu": 0.0000000128 // TODO check if this should be divided by 2
```

Comment:
I think if this number is for 64 bytes then it (and `kdf_time`) should be divided by 2.

Reply:
fixed